ykpack/src/types.rs

```diff
@@ -384,7 +388,9 @@ pub enum Terminator {
     },
     Call {
         operand: CallOperand,
-        ret_bb: Option<BasicBlockIndex>,
+        args: Vec<Operand>,
+        /// If the call converges, the return value and basic block to continue at.
```

Comment:
Sentence doesn't parse.

Reply:
Makes sense to me, but we can try a re-word.

How about:
```
`Some` if the call is convergent, `None` otherwise. When `Some` contains the return value and the block to return to.
```

Reply:
This also doesn't parse:
When `Some` contains the return value and the block to return to...what?

Reply:
I think it's the ending sentences on prepositions that confuses me.

Reply:
Okay I think I understand it now. It makes more sense to me if we just turn the sentence around: `The return value and basic block to continue at, if the call converges.`

Reply:
Fixed